### DISCUSSION TITLE
Bring back "compiling..." message

### DISCUSSION THREAD

**Discussion by edokan** [webpack/webpack#15391] [Q&A]
I'm using WebStorm, my colleagues use VSCode and run WebPack in a terminal with dev-server and hot-reload enabled. when we make changes to the files and save them, the file watcher catches the changes, says "compiling...", compiles, and then the browser reloads changed files, so far so good. But with newer versions, WebPackno longer shows the "compiling..." message. While it seems insignificant, we cannot see if the file watcher detected changes and started compiling or not, and with our build times getting longer and longer it causes a lot of frustration. Please bring back the good ol' message.

**Comment by alexander-akait** [comment]
Please create reproducible test repo and provide information about version and os

**Reply by edokan** [reply]
sure

**Comment by markjm** [comment]
FWIW, I have also brought back a (potentially more informative) message using the [compiler.hooks.invalid hook](https://webpack.js.org/api/compiler-hooks/#invalid). Something along the lines of ``Invalidated ${filename}`` printed to log. This acts both as a "Compiling..." message, and also helps identify errant recompiles due to other tools potentially modifying files behind your back.

IMO, this is also the best hook too capture the full rebuild time (changeTime - end of done hooks), to account for delay between `invalid` and `watchRun`

**Reply by alexander-akait** [reply]
Hm, we have that https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/plugins/CLIPlugin.js#L89
